src/components/AdvancedFilters/index.jsx

```diff
@@ -1,7 +1,63 @@
 import React from "react";
+import {
+  Container,
+  Button,
+  Accordion,
+  Card,
+  Tooltip,
+  OverlayTrigger,
+} from "react-bootstrap";
+import filters from "./filters";
+import "./index.css";
+import FilterOptionsList from "../FilterOptionsList";
+import { useTranslation } from "react-i18next";
 
 const AdvancedFilters = () => {
-  return <div></div>;
+  const { t } = useTranslation();
+
+  return (
+    <div className="advancedFilters">
+      <Container>
+        <div className="advancedFiltersTitleSection d-flex justify-content-between align-items-baseline pb-2">
+          <h5 className="advancedFiltersTitle">
+            {t("resources.AdvancedFilters.title")}
+          </h5>
+          <OverlayTrigger
+            overlay={
+              <Tooltip className="tooltipInfo">
+                {t("resources.AdvancedFilters.tooltipInfo")}
+              </Tooltip>
+            }
+          >
+            <span className="moreInfoFilters">?</span>
+          </OverlayTrigger>
+        </div>
+        {filters.map((filter) => {
+          return (
+            <Accordion key={filter.filterName}>
+              <Accordion.Toggle as={Card.Header} eventKey="0">
```

Comment:
what does this event key represent ? maybe we can change the name of the eventkey depending on the event

Reply:
starting value = 0 then the accordion is close

Reply:
but it comes from bootstrap so maybe if I change won't work

Reply:
yes, if I change the name it doesn't work